platform/android/MapboxGLAndroidSDKLint/.gitignore

```diff
@@ -1 +1,3 @@
 /build
+local.properties
+
```

Comment:
Did you delete the file in a different branch?

Reply:
Not intentionally.

Reply:
Ah OK then probably you can remove the file with this pull request.

Reply:
https://github.com/maplibre/maplibre-gl-native/blob/main/platform/android/MapboxGLAndroidSDKLint/local.properties is still on main...